docs/advanced/low-level-queries.md

```diff
@@ -0,0 +1,84 @@
+# Answering questions with custom queries
```

Comment:
Just as a note, StoryBook doesn’t like file names that don’t match the title (that’s why we have e.g. `transactionrequest.md` rather than `transaction-request.md`), but I don’t think it matters since we protected the `master` branch.